### PR TITLE
Ads 6242 price floors on auction end stable next base

### PR DESCRIPTION
**Pull request by hankyates-maven**
this pairs with https://github.com/themaven-net/Prebid.js/pull/66

### CODE REVIEW

**Comment by partha0solapurkar** [comment]
I see that the branches `stable` and `stable-next` have diverged in terms of commits but not in terms of the diff. I had fixed this earlier this year by merging `stable-next` into `stable` and then merging `stable` into `stable-next` after checking with @themaven-net/monetization . Thoughts on doing that again before they diverge too much?

**Comment by AritraSengupta** [comment]
@partha0solapurkar I think there were some changes from Brad that never made it to stable and it is blocked on something. Can you please follow up with him?

**Review by AritraSengupta — APPROVED** [review]
LGTM!

**Comment by partha0solapurkar** [comment]
Yeah, we can defer syncing them up if @bradchoate or anyone else has changes in progress. I think @elliotz1120 may also have changes in progress...

File: modules/mavenDistributionAnalyticsAdapter.js [hunk 1]
```diff
@@ -231,6 +235,7 @@ export function summarizeAuctionEnd(args, adapterConfig) {
   const bidStatusss = []
   const bidAmountss = []
   const bidResponseTimess = []
+  const floorss = []
```
Comment (HojunByun) [inline modules/mavenDistributionAnalyticsAdapter.js:238]: NIT: Array name could be confusing to store `floors`, suggest changing it to `floors_array`.

Comment (hankyates-maven) [reply modules/mavenDistributionAnalyticsAdapter.js:238]: I'm just keeping the convention consistent with the other fields

**Pull request merged**